app/services/product_drive_destroy_service.rb

```diff
@@ -0,0 +1,23 @@
+class ProductDriveDestroyService
+  def initialize(product_drive)
```

Comment:
Can we change `call` to a class method? It's easier to work with in terms of stubbing tests.